crates/parameters/src/lib.rs

```diff
@@ -474,3 +488,33 @@ where
 pub fn native_erc20_key() -> Key {
     storage::get_native_erc20_key_at_addr(ADDRESS)
 }
+
+/// Initialize parameters to the storage for testing
+#[cfg(any(test, feature = "testing"))]
```

Comment:
I believe `test` is not propagated to the dependency files and so we end up not finding this function